src/GlslCanvas.js

```diff
@@ -46,7 +47,7 @@ export default class GlslCanvas {
         this.vbo = {};
         this.isValid = false;
 
-        this.vertexString = options.vertexString || `
+        this.vertexString = contextOptions.vertexString || `
```

Comment:
`contextOptions` is not suitable here. `contextOptions` is used as the second argument in `canvas.getContext` so it seems that the `options` should be used here. The only concern is the backward compatibility - I suggest to swap `contextOptions` and `options` in the arguments.